libbpf-cargo/src/build.rs

```diff
@@ -78,61 +78,65 @@ fn check_clang(debug: bool, clang: &Path, skip_version_checks: bool) -> Result<(
 
 /// We're essentially going to run:
 ///
-///     clang -g -O2 -target bpf -c -D__TARGET_ARCH_$(ARCH) runqslower.bpf.c -o runqslower.bpf.o
+///   clang -g -O2 -target bpf -c -D__TARGET_ARCH_$(ARCH) runqslower.bpf.c -o runqslower.bpf.o
 ///
 /// for each prog.
-fn compile(debug: bool, objs: &[UnprocessedObj], clang: &Path) -> Result<()> {
+fn compile_one(debug: bool, source: &Path, out: &Path, clang: &Path) -> Result<()> {
```

Comment:
Would you consider adding an API to support custom compiler args as well? I have a use case where I'd like to pass -D and -I flags to clang when compiling my BPF object.

Reply:
Yeah, for sure. I added it as the 5th commit